src/components/pages/HomePage.jsx

```diff
@@ -1,28 +1,160 @@
+        <div className="lg:pt-36 pt-4 sm:pt-10 lg:text-left text-center items-center justify-center">
+          <h1 className="text-4xl sm:text-5xl md:text-6xl font-spartan font-semibold tracking-wide leading-tight pt-3 mb-8">
+            Helping unlock
+            <br />
+            your talent with
+            <br />
+            the best teachers
+          </h1>
+          <div className="flex justify-center lg:justify-start">
+          <button
+            className="flex items-center bg-red hover:bg-pureWhite hover:text-red hover:border-2 hover:border-red text-white font-spartan font-semibold text-2xl py-4 px-8 rounded-lg"
+            onClick={search}
+          >
+            Get started
+            <img src={ArrowSearch} alt="Arrow Right" className="ml-4 w-6 h-6" />
+          </button>
+          </div>
+         
+        </div>
+
+        <div className="m-10">
+          <img className="w-[700px]" src={mainImage} alt="art skills image" />
+        </div>
+      </section>
+      <section className="flex flex-col lg:flex-row items-center md:m-auto justify-center pt-2 lg:pt-16">
+        <div className="relative w-full lg:max-w-lg md:pb-12 pl-0">
+          <img
+            className="scale-90 md:scale-100 lg:scale-100 -ml-4 pb-0 md:pb-4 pl-0 lg:pl-0 md:pl-8"
+            src={GreenBlobImage}
+            alt="green blob"
+          />
+          <div
+            className="absolute top-1/4 left-1/8
+        "
+          >
+            <div className="text-2xl lg:text-3xl  md:text-3xl sm:text-2xl  font-spartan font-semibold text-white text-left pl-32 lg:pl-24">
+              Courses for
+              <br /> every age
+            </div>
+          </div>
+        </div>
+        <div className="relative  w-full lg:max-w-lg pb-10 md:pb-20">
+          <img
+            className="scale-100 md:scale-125 lg:scale-125"
+            src={YellowBlobImage}
+            alt="yellow blob"
+          />
+          <div className="absolute top-1/4 left-1/4  ml-1 lg:-ml-3">
+            <div className="text-2xl lg:text-3xl md:text-3xl sm:text-2xl font-spartan font-semibold text-white">
+              More than 100+
+              <br /> subject taught
+            </div>
+          </div>
+        </div>
+        <div className="relative  w-full lg:max-w-lg">
+          <img
+            className="scale-100 sm:scale-125 md:scale-125 lg:scale-125 pt-2 -mt-2 md:mt-4 lg:-mt-4"
+            src={PurpleBlobImage}
+            alt="purple blob"
+          />
+          <div className="absolute top-14 lg:top-10 left-1/4">
+            <div className="text-2xl lg:text-3xl md:text-3xl sm:text-2xl font-spartan font-semibold text-white text-left">
+              1000+
+              <br />
+              experienced teachers
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section className="relative py-10">
+        <div className="container mx-auto px-4">
```

Comment:
I think you need to create an array with data about courses, and then create elements using map method. It's better because if we need to change or add something we can change the data in the array. This is just an opinion, you can leave it as it is now.

Reply:
It's better to do this now. It'll also clean up the code and remove repetition. Great suggestion.

Reply:
got it! Thank you!

Reply:
We already have api for classes, maybe it is better approach to use it? We also have some data in out DB

Reply:
That's much better if you already have real data coming in.